src/swapi_client/v2/queryset_core.py

```
@@ -0,0 +1,145 @@
+        Zwraca CoreModel z dynamicznymi polami.
+        """
+
+        if not path.startswith("/"):
+            path = "/" + path
+
+        params = self._build_params()
+
+        try:
+            resp = await self.client.get(path, params=params)
+        except SWAPIError as e:
+            raise SWAPIError(f"GET {path} failed: {e}")
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)